data/announcements/scheduled.yaml

```
@@ -217,3 +217,16 @@ announcements:
     The legacy Linux package repositories (`apt.kubernetes.io` and `yum.kubernetes.io` AKA `packages.cloud.google.com`)<br/>
     have been frozen starting from September 13, 2023 **and are going away in January 2024**, users *must* migrate.<br/>
     Please read our [announcement](/blog/2023/08/31/legacy-package-repository-deprecation/) for more details.
+
+- name: Kubecon 2023 EU
+  startTime: 2024-02-16T00:00:00  #Added in https://github.com/kubernetes/website/pull/44828
+  endTime: 2024-02-22T18:00:00
```

End date should be March
```suggestion
  endTime: 2024-03-22T18:00:00
```